examples/docker-compose-rlcraft.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.8'
+
+services:
+  rlcraft:
+    image: itzg/minecraft-server:java8
+    container_name: rlcraft
+    volumes:
+    - rlcraft-modpack:/modpacks:ro
+    - rlcraft-data:/data
+    environment:
+      EULA: "true"
+      TYPE: "FORGE"
```

Comment:
TYPE needs to be "CURSEFORGE" to make use of CF_SERVER_MOD variable.

Reply:
Fixed in a1e4203